lms/services/lti_outcomes.py

```diff
@@ -129,22 +124,15 @@ def _canvas_request_modification(
         if submitted_at:
             request["submissionDetails"] = {"submittedAt": submitted_at.isoformat()}
 
-    @classmethod
-    def find_element(cls, xml_element, path):
-        """Extract element from LTI Outcomes Management XML response."""
-        xpath = "/".join([f"{{{cls.XML_NS}}}{name}" for name in path])
-
-        return xml_element.find(f".//{xpath}")
```

Comment:
Yeah. Pretty good to see this gone. WTF is `{{{`

Reply:
The inner one is a string format, then I think the next is an escape.

ElementTree munges the namespace into the tag name like:

 * `{namespace}tag`

Although the kicker here is you can call:

```
xml_element.find('//ns:tag/ns:another_tag', {'ns': "http:...." })
```